core/os/device/device.proto

```diff
@@ -56,6 +56,21 @@ enum OSKind {
   Stadia = 5;
 }
 
+// Incompatibility list possible reasons why a device is not compatible to
+// replay a given trace.
+enum Incompatibility {
+  Compatible = 0;  // Default, for compatible devices
+  ABIOS = 1;
+  ABIArchitecture = 2;
+  ABIMemoryLayout = 3;
+  GPUVendorID = 4;
+  GPUDeviceID = 5;
+  VkDriverVersion = 6;
+  VkAPIVersion = 7;
```

Comment:
Remove the 'Vk' part. Where we can, let's keep the gapis API graphics API agnostic.

Reply:
done